test/dnnl/TestApiBasic.cpp

```diff
@@ -1,29 +1,19 @@
-/*
- * Copyright (C) 2024 Intel Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions
- * and limitations under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
+//===-- TestApiBasic.cpp - Basic tests for the dnnl interface ---*- C++ -*-===//
```

Comment:
which license should be used for onednn related code in our repo? Added @kurapov-peter @ZhennanQin for discussion.

Reply:
Sorry, I missed this one. These are not part of llvm, so should have the copyright.